README.md

```diff
@@ -1,101 +1,82 @@
-# HTML & CSS3 & JavaScript Course
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-If you are not familiar with linters, read [root level README](../README.md).
+# Music Extravaganza
 
-## Set-up GitHub Actions
+> "Music Extravaganza" is a Microverse requirment to complete module one of the program. The project is about musicians showcasing their talent.
+> The project consists of two pages:
+- Home page
+- About page
 
-Please do the following **steps in this order**:
+![screenshot](./images/top.png)
+![screenshot](./images/singers.png)
+![screenshot](./images/programs.png)
 
-1. In the first commit of your feature branch create a `.github/workflows` folder and add a copy of [`.github/workflows/linters.yml`](.github/workflows/linters.yml) to that folder.
-   - **Remember** to use the file linked above
-   - **Remember** that `.github` folder starts with a dot.
-2. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
-   - If you think that change is necessary - open a [Pull Request in this repository](../README.md#contributing) and let your code reviewer know about it.
-3. When you open your first pull request you should see the result of the GitHub actions checks:
+- The design if from [Cindy Shin](https://www.behance.net/adagio07) The design its self can be viewed [here](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
 
-![gh actions checks](../assets/images/gh-actions-html-css-checks.png)
+## Built With
 
-Click on the `Details` link of each action to see the full output and the errors that need to be fixed:
+- HTML-5
+- CSS3
+- Javacript
+- Linters
+- Google font awesome icons
 
-![gh actions failing checks](../assets/images/gh-actions-html-css-failing-checks.png)
+## Live Demo
+Live site URL
```

Comment:
- [ ] Good job so far 👍 but you are supposed to deploy your site using Github pages and provide the live demo link in your README but the link you have provided is that for your repo. It would be great if you could provide the right link.